<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandmarkDialog</class>
 <widget class="QDialog" name="LandmarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Landmark editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="viewLandmark"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemoveAll">
       <property name="text">
        <string>Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCloseView">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonCloseView</tabstop>
  <tabstop>viewLandmark</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCloseView</sender>
   <signal>clicked()</signal>
   <receiver>LandmarkDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
